community/sgt-puzzles/build

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh -e
+
+./configure \
+    --prefix=/usr \
+    --with-gtk=3
+make
+
+make DESTDIR="$1" install
+
+# Prefix binaries to avoid name conflicts with unrelated packages
```

Comment:
You can use --program-prefix=sgt during ./configure, no?

Reply:
omg I didn't know that existed!  I was going off of the arch PKGBUILD; I guess whoever wrote that didn't know about it either haha

Reply:
I'm convinced most people who write PKGBUILDs don't ever check ./configure --help

Reply:
well fwiw I do read ./configure --help , just not the whole thing lol